lib/nodes/image.js

```diff
@@ -0,0 +1,41 @@
+import placeholder from "../assets/placeholder.jpg";
+
+export const name = "image";
+
+export function schema(options) {
```

Comment:
I think long term we'd probably want to parse the options out higher up the stack before we start building schema and then just pass in the options that are relevant to this node type, but this'll do for now